palisade-service/src/main/resources/application-eureka.yaml

```diff
@@ -36,18 +36,19 @@ eureka:
     port: 8083
     preferIpAddress: false
     lease-renewal-interval-in-seconds: 5
+    metadataMap:
+      instanceId: ${vcap.application.instance_id:${spring.application.name}:${spring.application.instance_id:${server.port}}}
   client:
     serviceUrl:
       defaultZone: http://${eureka.instance.hostname}:${eureka.instance.port}/eureka/
     enabled: true
     fetchRegistry: true
     registry-fetch-interval-seconds: 5
 
-# Eureka discovery by lookup for registered service by name
-# i.e. map from what-i-call-a-service -> what-that-service-calls-itself
+# Discovery by spring.application.name
 web:
   client:
-    audit-service: audit-service
-    policy-service: policy-service
-    resource-service: resource-service
-    user-service: user-service
+    audit-service: ""
+    policy-service: ""
+    resource-service: ""
+    user-service: ""
```

Comment:
I would have expected these to be as before. if we do want them to be undefined can we make it clear in the name that they are undefined, e.g. palisade-service-not-configured

Reply:
These are actually the values they need to be in order to work correctly. Unfortunate that it is.